app/utils/zip-agenda-files.js

```diff
@@ -18,17 +18,27 @@ async function constructArchiveName(agenda) {
 
 async function fetchArchivingJob(agenda) {
   const url = `/agendas/${agenda.id}/agendaitems/documents/files/archive`;
-  let job = (await fetch(url, {
+  return await fetch(url, {
     method: 'post',
     headers: { 'Content-type': 'application/vnd.api+json' }
-  })).json();
-  return job;
+  }).then((fetchedJob) => {
```

Comment:
Plz use `await` combined with `try ... catch` in async functions :pray: :-)

Reply:
Fixed it by removing the .then .catch structure, try catch is not needed in this case